Dockerfile

```
@@ -0,0 +1,11 @@
+FROM node:14
+
+WORKDIR /app
+
+COPY package*.json ./
+
+RUN npm install
+
+EXPOSE 3000
+
+CMD ["npm", "run", "start"]
```

На первый взгляд не уверен, что это корректный файл. По-моему мнению должно быть что-то типа 
```suggestion
FROM node:14

WORKDIR /app

COPY package*.json ./

RUN npm install

COPY . .

EXPOSE 3000

CMD npm run start
```